app/controllers/items_controller.rb

```diff
@@ -5,6 +5,7 @@ def browse
   end
 
   def index
+    @extra_js = [javascript_include_tag("search")]
```

Comment:
if we wanted to avoid the `html_safe` business, presumably we could run the `javascript_include_tag` on any strings passed in from this array below.

```ruby
 <% if !@extra_js.blank? %>
  <% Array.wrap(@extra_js).each do |js|
    javascript_include_tag(js)
  end %>
<% end %>

or something like that?